lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/client/YarnClientConfiguration.java

```diff
@@ -50,6 +51,7 @@
   public static final OptionalParameter<Integer> YARN_PRIORITY = new OptionalParameter<>();
 
   public static final OptionalParameter<Double> JVM_HEAP_SLACK = new OptionalParameter<>();
+  public static final OptionalParameter<Boolean> YARN_FEDERATION = new OptionalParameter<>();
```

Comment:
Please add JavaDoc to this. The documentation should include the links to the relevant JIRAs.